draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -758,7 +756,7 @@ checks the Disclosures correctly.
 
 ## Entropy of the salt {#salt-entropy}
 
-The security model relies on the fact that the salt is not learned or guessed by
+The security model that conceals the cleartext claims relies on the fact that the salt cannot be learned or guessed by
```

Comment:
```suggestion
The security model that conceals the plaintext claims relies on the fact that the salt cannot be learned or guessed by
```

Reply:
The draft currently uses plaintext, cleartext and plain text. Maybe a full reconciliation is needed at some point. In the meantime, I'll take this suggestion.